tutorials/tour/basics.md

```diff
@@ -90,33 +91,33 @@ println({
 
 ## Functions
 
-Functions are expressions with parameters.
+Functions are expressions that take parameters and compute a result.
```

Comment:
Ummm... "expressions with parameters" just sounds funny to me. I would never say/write that. Does it mean "functions are values", or does it mean that the body of a function is an expression, or what?

I'm open to other suggestions than the one I've made, but I do think the original sentence is unclear and should be changed _somehow_.